sidebars.js

```diff
@@ -63,18 +96,58 @@ const sidebars = {
           label: 'SDKs',
           collapsible: true,
           items: [
-            'scalardl-java-client-sdk/README',
+            {
+              type: 'doc',
+              id: 'scalardl-java-client-sdk/README',
+              label: 'ScalarDL Java Client SDK',
+            },
           ],
         },
-        'configurations',
-        'schema-loader',
-        'ca/caclient-getting-started',
-        'authentication',
-        'how-to-handle-errors',
-        'how-to-write-contract',
-        'how-to-write-function',
-        'use-generic-contracts',
-        'how-to-use-proof',
+        {
+          type: 'doc',
+          id: 'configurations',
+          label: 'ScalarDL Configurations',
+        },
+        {
+          type: 'doc',
+          id: 'schema-loader',
+          label: 'ScalarDL Schema Loader',
+        },
+        {
+          type: 'doc',
+          id: 'ca/caclient-getting-started',
+          label: 'Get a Certificate',
+        },
+        {
+          type: 'doc',
+          id: 'authentication',
+          label: 'Authentication',
+        },
+        {
+          type: 'doc',
+          id: 'how-to-handle-errors',
+          label: 'Handle Errors',
+        },
+        {
+          type: 'doc',
+          id: 'how-to-write-contract',
+          label: 'Write a Contract',
+        },
+        {
+          type: 'doc',
+          id: 'how-to-write-function',
+          label: 'Write a Function',
+        },
+        {
+          type: 'doc',
+          id: 'use-generic-contracts',
+          label: 'Use Generic Contracts and Functions',
```

Comment:
@jnmt As mentioned in [your comment for the Japanese phrasing](https://github.com/scalar-labs/docs-scalardl/pull/597#discussion_r1893676750), should we shorten the English wording as well?


```suggestion
          label: 'Use Generic Contracts',
```

Reply:
@josh-wong Yes, we should do the same.